### ISSUE TITLE
Collapsible side bar, search menu on top

### ISSUE THREAD

**Issue by ayushi-jain97** [coala/landing-frontend#58]
A collapsible side bar with a toggle menu on the left with a search bar at the top. Coala logo can be shifted inside the collapsible bar. A search menu on the Home page is a must.
 | Logo |
-Home
-Languages
-Get Involved
-Try Coala
-DOCS
-Source
-Releases
Contact info

**Comment by sils** [comment]
I'm not sure I get the reasoning why this needed, also not sure how you want it to look. The current design is very plain and nice to look at while being rather minimal and forcing us to keep it minimal.

**Comment by ayushi-jain97** [comment]
Yes you are right. The design is simple and easy going but there is lack of
connectivity between pages.

For eg - We cannot reach the DOCS directly from Getting Involved section or
vice versa.

Thanking you
Yours sincerely
Ayushi Jain
Sophomore
Dept of CSE
IIT-BHU, Varanasi
India
